scripts/gcov.sh

```
@@ -0,0 +1,19 @@
+
+# Before running this script, make sure $VCPKG_ROOT is set, e.g.
+# VCPKG_ROOT="$HOME"/vcpkg && export VCPKG_ROOT
+
+cd ..
+rm -rf build/
+cmake -G Ninja -D ENABLE_COVERAGE:BOOL=TRUE -D CMAKE_BUILD_TYPE=Debug -D ENABLE_TESTING:BOOL=TRUE -S . -B build
+cmake --build build
+cd build || exit
+ctest --output-on-failure -j2
+mkdir gcov-reports
+pushd gcov-reports
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)